packages/core/src/services/time-series.ts

```diff
@@ -0,0 +1,140 @@
+import { format } from "date-fns";
+import * as locales from "date-fns/locale";
```

Comment:
This will add to the bundle size of the library. If we don't directly use this piece we should take it out and switch

`throw new Error('Locale with code ${localeCode} not found. Avaible codes are: ${codes}.');` to just include a link to a page that lists all the codes

Reply:
Ok, I'll work on it!